sources/packages/web/src/views/aest/ApplicationDetails.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Student Details</a
+    >
+  </h5>
+  <full-page-container v-if="applicationDetail.data">
+    <h2 class="color-blue padding-bottom-30">
+      Student Application Details
+      {{
+        applicationDetail.applicationNumber
+          ? " - " + applicationDetail.applicationNumber
+          : ""
+      }}
+    </h2>
+    <formio
+      :formName="selectedForm"
```

Comment:
How the cascading dropdowns are being loaded?

Reply:
They are not loading. #761 has been created to address the Student Application dropdown fix and organize the APIs after discussing with @abschwenker

Reply:
Just to be clear, the ticket is not about fixing dropdowns, it is about creating a common component to be shared among the Student and the Ministry portal, that yes, will fix the dropdowns, but also fixed everything else that is needed 😉